### PR TITLE
Add more flexible definition of the IW crystal

### PR DESCRIPTION
**Pull request by Laar**
Temporary work till #61 is fixed, a new definition of the inverse woodpile similar to that of structure 4.

I did not replace structure 4 as that can be kept to match the original meshes with a slightly different design.

### CODE REVIEW

File: applications/DG-Max/ElementInfos.cpp [hunk 1]
```diff
@@ -159,6 +200,26 @@ double jelmerStructure(const Geometry::PointPhysical<DIM>& pPhys,
         // std::cout << pPhys[0] << " " << pPhys[1] << " " << pPhys[2]  << "
         // " << epsilon_ << "\n";
 
+    } else if (structureType == 5) {
```
Comment (JensWehner) [inline applications/DG-Max/ElementInfos.cpp:203]: can we at least make the structure Type an enum?

**Review by JensWehner — APPROVED** [review]
LGTM

File: applications/DG-Max/ElementInfos.cpp (hunk 1 above)
Comment (Laar) [reply applications/DG-Max/ElementInfos.cpp:203]: Should have known this comment was coming, I'll see what I can do. Problem is that the input goes via a command line argument.

Comment (Laar) [reply applications/DG-Max/ElementInfos.cpp:203]: Done

**Pull request merged**